### DISCUSSION TITLE
Pressure sensitive buttons through DS4Windows?

### DISCUSSION THREAD

**Discussion by InterClaw** [nefarius/DsHidMini#289] [Q&A] [answered]
Would the DS3 pressure sensitive buttons work when used with DS4Windows? I'm guessing no, since DS4Windows only outputs as a DS4 or 360 controller, right?

**Comment by nefarius** [comment, marked as answer]
Since neither the Xbox 360 nor the DS4 controller have any concept of pressure sensitive buttons, nope.
